client/src/components/ElementActions/AbstractAction.js

```diff
@@ -30,11 +31,13 @@ AbstractAction.propTypes = {
   name: PropTypes.string,
   type: elementTypeType,
   active: PropTypes.bool,
-  label: PropTypes.string
+  label: PropTypes.string,
+  showForNonInlineEditableBlock: PropTypes.bool
```

Comment:
Not a big fan of the wording here. Could we rename this to something like `alwaysShow` or `showOnlyForInlineEditable`.